concrete/src/Routing/RoutingServiceProvider.php

```diff
@@ -10,7 +10,7 @@ class RoutingServiceProvider extends Provider
      */
     public function register()
     {
-        $this->app->singleton(RouterInterface::class);
+        $this->app->singleton(Router::class);
         $this->app->bind(RouterInterface::class, Router::class);
```

Comment:
Let's suppose the core code is
```php
$this->app->singleton(Router::class);
$this->app->alias(Router::class, RouterInterface::class);
```
If after that we set
```php
$this->app->alias(CustomRouter::class, RouterInterface::class);
```
then this should be true:
```php
$instance1 =  $this->app->make(RouterInterface::class);
$instance2 =  $this->app->make(RouterInterface::class);
$instance3 =  $this->app->make(CustomRouter::class);
$instance4 =  $this->app->make(CustomRouter::class);
($instance1 === $instance2 && $instance2 === $instance3 && $instance3 === $instance4);
```

but obviously if we call `$this->app->make(Router::class);` we have a different instance (it'll be a `Router` instance).
I don't see any problem, given that we call `$this->app->make(RouterInteface::class);` to get the currently cofigured router.